### ISSUE TITLE
"track user!" popup repeatedly after visiting their public dir

### ISSUE THREAD

**Issue by imnotpete** [keybase/keybase-issues#2386]
I visited the public dir of a user I don't track, to see what it would show me. Now the app gives me a popup about that user with a "track" button at the bottom, a couple of times per day. I can "x" out the screen, but there's no "leave me alone" button anywhere to get it to stop prompting me.

Windows 10

**Comment by strib** [comment]
You can try to `del` the folder from the command line to remove it from your list of favorites.  Then Windows should stop being able to see it -- though I'm not sure exactly how Windows does its indexing; if  the indexer/spiderer remembers the folder it might still be able to look it up again and re-add it to your favorites again.

**Comment by imnotpete** [comment]
Deleting from either Explorer or the command line both seem to have no effect. I tried the folder a couple days ago and it didn't change. I've just tried `del` and the folder's still there -- I'll keep an eye out for the pop up tomorrow.

**Comment by strib** [comment]
Hmm I thought that was supposed to delete the folder and not show it to you anymore.  @taruti do we still support favorite removal on Windows?

**Comment by strib** [comment]
@imnotpete: a `keybase log send` from the keybase cmd shell might be helpful.

**Comment by imnotpete** [comment]
@strib done, with a warning.

log id: a470a32ddd502e6863830f1c
warning: 
`- WARNING error opening log "C:\\Users\\sam\\AppData\\Roaming\\Keybase\\keybase.start.log": open C:\Users\sam\AppData\Roaming\Keybase\keybase.start.log: The system cannot find the file specified.`

**Comment by strib** [comment]
Oh, maybe it's not `del`.  Try `rmdir`?  (Sorry, I'm don't have a Windows box handy for testing.)

**Comment by imnotpete** [comment]
Ah. Yeah, that at least removed it from my view. I'll see if I get a popup tomorrow. Thanks!

**Comment by imnotpete** [comment]
Nope, got the popup this morning and the folder is back in `K:\public`

**Comment by taruti** [comment]
Tried it today with both `rmdir` and explorer on staging and it worked.

@imnotpete 
Is it possible that Windows or some invidual program has a "recent files" or similar functionaly referring to a file inside that shared folder? And tries to open it and thus resubscribes you?

**Comment by strib** [comment]
@imnotpete: sorry about that.  As a workaround of course you can track the user.  If you don't want to track them publicly, the command line has the ability to only track locally (`keybase track -l`).
